ims.yaml

```diff
@@ -0,0 +1,672 @@
+          schema:
+            $ref: '#/definitions/PersonSetResult'
+        '400':
+          $ref: '#/responses/400'
+        '401':
+          $ref: '#/responses/401'
+        '403':
+          $ref: '#/responses/403'
+        '429':
+          $ref: '#/responses/429'
+        '500':
+          $ref: '#/responses/500'
+  /persons/{id}:
+    get:
+      operationId: getPerson
+      summary: The REST read request message for the getPerson() API call.
+      tags:
+        - PersonsManagement
+      description: ...not supplied...
+      parameters:
+        - name: id
+          in: path
+          description: ...tbd...
+          required: true
+          type: string
+        - name: fields
+          in: query
+          description: To identify the range of fields that should be supplied in the response message.
+          required: false
+          type: array
+          items:
+            type: string
+          collectionFormat: csv
+          allowEmptyValue: false
+      responses:
+        '200':
+          description: The request was successfully completed and a record has been returned. This would be accompanied by the 'codeMajor/severity' values of 'success/status' and for a REST binding a HTTP code of '200'.
+          schema:
+            $ref: '#/definitions/PersonResult'
+        '400':
+          $ref: '#/responses/400'
+        '401':
+          $ref: '#/responses/401'
+        '403':
+          $ref: '#/responses/403'
+        '404':
+          $ref: '#/responses/404'
+        '422':
+          $ref: '#/responses/422'
+        '429':
+          $ref: '#/responses/429'
+        '500':
+          $ref: '#/responses/500'
+    delete:
+      operationId: deletePerson
+      summary: The REST delete request message for the deletePerson() API call.
+      tags:
+        - PersonsManagement
+      description: ...not supplied...
+      parameters:
+        - name: id
+          in: path
+          description: ...tbd...
+          required: true
+          type: string
+      responses:
+        '204':
+          description: ...tbd (Response description)...
+        '400':
+          $ref: '#/responses/400'
+        '401':
+          $ref: '#/responses/401'
+        '403':
+          $ref: '#/responses/403'
+        '404':
+          $ref: '#/responses/404'
+        '422':
+          $ref: '#/responses/422'
+        '429':
+          $ref: '#/responses/429'
+        '500':
+          $ref: '#/responses/500'
+    patch:
+      operationId: patchPerson
+      summary: The REST create request message for the patchPerson() API call.
+      tags:
+        - PersonsManagement
+      description: ...not supplied...
+      parameters:
+        - name: id
+          in: path
+          description: ...tbd...
+          required: true
+          type: string
+        - name: person
+          in: body
+          description: ...tbd...
+          required: true
+          schema:
+            $ref: '#/definitions/PersonResource'
```

Comment:
I think we should have a separate patch body in `definitions` since usually, this is a subset of the resource and makes all attributes optional